### ISSUE TITLE
 Vision not working 

### ISSUE THREAD

**Issue by Pascal66** [4mbilal/AlJazari#1]
seems not implemented

**Comment by 4mbilal** [comment]
> seems not implemented

Yes, the project update is in the works. Please checkout an older version from the history (2019). You can find it there.

**Comment by Pascal66** [comment]
Thank you, i'll check there
Ok, it work.

**Comment by Pascal66** [comment]
Vision controller work (my interest)
What's the goal of Night vision with a canny ?
Object detection doesnt work
